libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -146,6 +152,20 @@ void AP_Vehicle::setup()
     // init_ardupilot is where the vehicle does most of its initialisation.
     init_ardupilot();
 
+#if AP_AIRSPEED_ENABLED
+    airspeed.init();
+    if (airspeed.enabled()) {
+        // initialize airspeed sensor
+        // --------------------------
+        airspeed.calibrate(true);
+    } 
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+    else {
+        gcs().send_text(MAV_SEVERITY_WARNING,"No airspeed sensor present");
```

Comment:
```suggestion
        GCS_SEND_TEXT(MAV_SEVERITY_WARNING,"No airspeed sensor present");
```

I wonder if we can just omit this message now.  It's not even correct - the thing could be present but not enabled :-)

Still, don't want to change behaviour on Plane, so...

Just wish we could avoid the `else` dangling in the `#if`.

Reply:
Done on the change. 

I was assuming I'd get a comment about the #if ;) but.... this PR is getting long in the tooth and this seems like future cleanups.